osu.Game/Configuration/OsuConfigManager.cs

```diff
@@ -20,6 +20,8 @@ protected override void InitialiseDefaults()
             Set(OsuSetting.DisplayStarsMinimum, 0.0, 0, 10);
             Set(OsuSetting.DisplayStarsMaximum, 10.0, 0, 10);
 
+            Set(OsuSetting.SelectionRandomType, SelectionRandomType.Random);

```

Comment:
I think it's a better default, but let's leave the setting for the time being. Since we can easily add more settings (and will add more filtering/grouping of the settings menu in the future) I see no harm in doing so.

Please change the default to this algorithm, though.